mirascope/cli/utils.py

```diff
@@ -292,20 +292,22 @@ def _update_tag_decorator_with_version(
     return import_name
 
 
-def _update_mirascope_imports(imports: list[str]):
+def _update_mirascope_imports(imports: list[tuple[str, Optional[str]]]):
     """Updates the mirascope import."""
-    if not any(import_name == "mirascope" for import_name in imports):
-        imports.append("mirascope")
+    if not any(import_name == "mirascope" for import_name, _ in imports):
+        imports.append(("mirascope", None))
 
 
-def _update_mirascope_from_imports(member: str, from_imports: list[tuple[str, str]]):
+def _update_mirascope_from_imports(
```

Comment:
There are two ` # TODO: Update `mirascope.tags` work with import alias` that need to be updated as well.

This use case needs to be supported: 
```
import mirascope as mira

@mira.tags()
class MyPrompt(Prompt):
    ...
```

Reply:
Right now the strings that we are checking are hardcoded to be `mirascope.tags`